sky/backends/backend_utils.py

```diff
@@ -768,7 +768,16 @@ def write_cluster_config(
 
     yaml_path = _get_yaml_path_from_cluster_name(cluster_name)
 
-    # Use a tmp file path to avoid incomplete YAML file being re-used in the future.
+    login_user = None
+    try:
+        login_user = getpass.getuser()
+    except Exception:  # pylint: disable=broad-except
+        # https://docs.python.org/3/library/getpass.html#getpass.getuser
+        # did not specify what exceptions will be raised if all attempts fail.
+        pass
```

Comment:
We are using this widely in the backend. If we would like to catch the exceptions, I think it would be better to have a wrapper function that does this and call the function in every other place as well.

Reply:
Decided to not catch. It should work for most OS except the esoteric ones. GitHub code search reveals `getuser()` is assumed to not raise: https://github.com/search?q=getpass.getuser&type=code